Tests/RingPublishingTrackingTests/DictionaryTests.swift

```diff
@@ -24,16 +24,4 @@ class DictionaryTests: XCTestCase {
         XCTAssertEqual(emptyDictionary.jsonSizeInBytes, 2, "Size in bytes should be correct")
         XCTAssertEqual(dictionaryWithData.jsonSizeInBytes, 31, "Size in bytes should be correct")
     }
-
-    func testJsonSizeInBytes_invalidDictionaryCreated_returnedSizeInBytesIsZero() {
```

Comment:
powiedz tylko czemu usuwamy to?

Reply:
Tu był test na invalid string w JSONie. String ten był tworzony:
`        let specialString = String(bytes: [0xD8, 0x00] as [UInt8],
                                   encoding: String.Encoding.utf16BigEndian)
`
Musiało się coś zmienić w Swifcie, że teraz specialSrting jest nil, a kiedyś był jakim dziwnym znakiem, który psuł json. Teraz przypisanie nil do klucza w słowniku nie spowoduje zepsucia jsona